examples/controls/webpack.config.js

```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```

Comment:
Wouldn't it be great if there was a comment in front of each line in our webpack configs that explained why a particular configuration is needed?

Particularly in this case it could save someone else a lot of time later.